invokeai/app/invocations/fields.py

```diff
@@ -242,6 +242,23 @@ class ConditioningField(BaseModel):
     )
 
 
+class BoundingBoxField(BaseModel):
```

Comment:
Suggest adding an ["after" model validator](https://docs.pydantic.dev/dev/concepts/validators/#model-validators) to prevent stuff like `x_max < x_min`

Reply:
Technically they can be though. Coz they're just coordinates of a two vertices of a box, it doesn't matter where they are as long as they are within the image bounds. It might lead to some wonky ass polygons, but still legit.

Reply:
it's common to do `x_max - x_min` to get `width`, which would result in a negative width